config_test.go

```diff
@@ -77,6 +113,24 @@ leds:
 			Recording: "RecordingPIN",
 			Running:   "RunningPIN",
 		},
+		Turret: TurretConfig{
+			Active: true,
+			PID:    []float64{1, 2, 3},
+			ServoX: ServoConfig{
+				Active:   false,
+				Pin:      "pin",
+				MaxAng:   180,
+				MinAng:   0,
+				StartAng: 30,
+			},
+			ServoY: ServoConfig{
+				Active:   true,
+				Pin:      "pin",
+				MaxAng:   180,
+				MinAng:   0,
+				StartAng: 30,
```

Comment:
Nice. It might be a good idea to use different numbers for X and Y to ensure they're being read into the correct fields.